rs/protobuf/def/state/queues/v1/queues.proto

```diff
@@ -178,14 +159,11 @@ message CanisterQueue {
 }
 
 message CanisterQueues {
-  reserved 1, 4;
-  reserved "canister_id", "input_schedule";
+  reserved 1, 3, 4, 5;
+  reserved "canister_id", "input_queues", "input_schedule", "output_queues";
```

Comment:
IIRC you left a nit on one of my PR's to move this to the bottom for [Stream](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/protobuf/def/state/queues/v1/queues.proto?L47).

Reply:
Done.

Thanks for keeping me honest. (o: